### PR TITLE
Update seqscreen

### PR DESCRIPTION
**Pull request by bkille**
Update the seqscreen tool to include a new pipeline. 

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/seqscreen/run_test.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+seqscreen --version
```
Comment (bgruening) [inline recipes/seqscreen/run_test.sh:1]: can you include this in the commands section in the meta.yml file?

Comment (bkille) [reply recipes/seqscreen/run_test.sh]: @bgruening This has caused issues in the past. For example, this time the command completes with a successful exit code but [the build fails](https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/33207/workflows/3d046e6e-ee17-4e70-aebe-f7d6dc2ab6e7/jobs/124628) with the following:

```
18:15:05 BIOCONDA INFO (ERR) [Sep 23 18:15:05] SERR See 'conda init --help' for more information and options.
18:15:05 BIOCONDA INFO (ERR) [Sep 23 18:15:05] SERR
18:15:05 BIOCONDA INFO (ERR) [Sep 23 18:15:05] SERR IMPORTANT: You may need to close and restart your shell after running 'conda init'.
18:15:05 BIOCONDA INFO (ERR) [Sep 23 18:15:05] SERR
18:15:05 BIOCONDA INFO (ERR) [Sep 23 18:15:05] SERR
18:15:09 BIOCONDA INFO (ERR) [Sep 23 18:15:09] DEBU Container [6280b83158da step-9aebe8a8c7] completed with exit code [0] as expected
18:15:09 BIOCONDA INFO (ERR) [Sep 23 18:15:09] DEBU Container [6280b83158da step-9aebe8a8c7] removed
18:15:09 BIOCONDA INFO (ERR) [Sep 23 18:15:09] INFO Invoke Task [test]
18:15:09 BIOCONDA INFO (ERR) [Sep 23 18:15:09] STEP Run image [quay.io/biocontainers/seqscreen:1.4.0--0] with command [[seqscreen --version]]
18:15:09 BIOCONDA INFO (ERR) [Sep 23 18:15:09] DEBU Creating container [step-c543b6617d]
18:15:09 BIOCONDA INFO (ERR) [Sep 23 18:15:09] DEBU Created container [263025f469c3 step-c543b6617d], starting it
18:15:09 BIOCONDA INFO (ERR) [Sep 23 18:15:09] DEBU Container [263025f469c3 step-c543b6617d] started, waiting for completion
18:15:12 BIOCONDA INFO (ERR) [Sep 23 18:15:12] SOUT N E X T F L O W  ~  version 20.07.1
18:15:12 BIOCONDA INFO (ERR) [Sep 23 18:15:12] SOUT Launching `/usr/local/bin/seqscreen` [tender_morse] - revision: 08b59dc357
18:15:13 BIOCONDA INFO (ERR) [Sep 23 18:15:13] SOUT SeqScreen v1.4.0
18:15:13 BIOCONDA INFO (ERR) [Sep 23 18:15:13] DEBU Container [263025f469c3 step-c543b6617d] completed with exit code [0] as expected
18:15:13 BIOCONDA INFO (ERR) [Sep 23 18:15:13] DEBU Container [263025f469c3 step-c543b6617d] removed
.18:15:32 BIOCONDA INFO (OUT) ....

Traceback (most recent call last):
  File "/home/circleci/project/miniconda/bin/bioconda-utils", line 8, in <module>
    sys.exit(main())
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/cli.py", line 971, in main
    bioconductor_skeleton, clean_cran_skeleton, autobump, bot
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/argh/dispatching.py", line 328, in dispatch_commands
    dispatch(parser, *args, **kwargs)
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/argh/dispatching.py", line 174, in dispatch
    for line in lines:
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/argh/dispatching.py", line 277, in _execute_command
    for line in result:
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/argh/dispatching.py", line 260, in _call
    result = function(*positional, **keywords)
  File "<boltons.funcutils.FunctionBuilder-5>", line 2, in build
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/cli.py", line 130, in wrapper
    func(*args, **kwargs)
  File "<boltons.funcutils.FunctionBuilder-4>", line 2, in build
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/cli.py", line 59, in wrapper
    func(*args, **kwargs)
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/cli.py", line 476, in build
    keep_old_work=keep_old_work)
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/build.py", line 343, in build_recipes
    linter=linter)
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/build.py", line 149, in build
    pkg_test.test_package(pkg_path, base_image=base_image)
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/bioconda_utils/pkg_test.py", line 170, in test_package
    p = utils.run(cmd, env=env, cwd=d, mask=False)
  File "/home/circleci/project/miniconda/lib/python3.7/tempfile.py", line 805, in __exit__
    self.cleanup()
  File "/home/circleci/project/miniconda/lib/python3.7/tempfile.py", line 809, in cleanup
    _shutil.rmtree(self.name)
  File "/home/circleci/project/miniconda/lib/python3.7/shutil.py", line 485, in rmtree
    _rmtree_safe_fd(fd, path, onerror)
  File "/home/circleci/project/miniconda/lib/python3.7/shutil.py", line 423, in _rmtree_safe_fd
    _rmtree_safe_fd(dirfd, fullname, onerror)
  File "/home/circleci/project/miniconda/lib/python3.7/shutil.py", line 443, in _rmtree_safe_fd
    onerror(os.unlink, fullname, sys.exc_info())
  File "/home/circleci/project/miniconda/lib/python3.7/shutil.py", line 441, in _rmtree_safe_fd
    os.unlink(entry.name, dir_fd=topfd)
PermissionError: [Errno 13] Permission denied: 'history'

Exited with code exit status 1
CircleCI received exit code 1
```

For some reason this does not happen when the command is in the `run_test.sh`

**Comment by bkille** [comment]
@BiocondaBot please add label

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Pull request merged**